src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  justify-content: center;
+  background-position: center;
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 :nth-child(n),
+.stars--1 :nth-last-child(-n + 4),
+.stars--2 :nth-last-child(-n + 3),
+.stars--3 :nth-last-child(-n + 2),
+.stars--4 :nth-last-child(-n + 1) {
+  background-image: url(images/star.svg);
+}
```

Comment:
```suggestion

```

Reply:
i'm fixed code but who to update?

Reply:
Not fixed. Consider pushing your changes here